app/main.py

```
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split(" ")
+    if len(command_split) != 3 or command_split[0] != "cp":
+        return
+
+    file_name = command_split[1]
+    new_file_name = command_split[2]
+
+    if file_name == new_file_name:
+        return
```

Similar to the previous comment, it would be helpful to provide feedback to the user if the source and destination filenames are the same. This can prevent silent failures and improve user experience.